consensus/src/consensus.rs

```diff
@@ -41,25 +49,15 @@ pub struct ConsensusClient<R: ConsensusRpc, DB: Database> {
 
 #[derive(Debug)]
 pub struct Inner<R: ConsensusRpc> {
```

Comment:
Why have all of these fields as well as many methods on this struct been made public when they don't seem to need to be?

Reply:
I reverted some fields to be private, but kept `store` and `rpc` public alongside the methods. These values and methods are useful inside other programs when helios is used as a library. 

e.g in sp1-telepathy we pass grab `store` and call `expected_current_slot` from `Inner`

https://github.com/succinctlabs/sp1-telepathy/blob/1fe903bd1507010180f303a2ca3be6ebbd4155ee/script/bin/operator.rs#L184-L189